ocaml/libs/uuid/uuidx.ml

```diff
@@ -83,3 +83,15 @@ let string_of_uuid = to_string
 let uuid_of_int_array = of_int_array
 
 let int_array_of_uuid = to_int_array
+
+module XenServer = struct
```

Comment:
This name is not really appropriate, as this is Xapi Project code, not exclusive to XenServer. Something related to the actual use case may be better?